<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetNetworkWindow</class>
 <widget class="QDialog" name="SetNetworkWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>186</width>
    <height>174</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Config. Rede</string>
  </property>
  <widget class="QLabel" name="Ip_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>IP</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Ip_lineEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>192.168.1.100</string>
   </property>
  </widget>
  <widget class="QLabel" name="port_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Port</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="port_lineEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>80</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Configurações de Rede</string>
   </property>
  </widget>
  <widget class="QLabel" name="error_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Ok_Button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Cancel_Button">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
